winml/lib/Api.Ort/OnnxruntimeEngineBuilder.cpp

```diff
@@ -77,4 +82,9 @@ STDMETHODIMP OnnxruntimeEngineBuilder::GetID3D12CommandQueue(ID3D12CommandQueue*
 STDMETHODIMP OnnxruntimeEngineBuilder::SetBatchSizeOverride(uint32_t batch_size_override) {
   batch_size_override_ = batch_size_override;
   return S_OK;
+}
+
+STDMETHODIMP OnnxruntimeEngineBuilder::SetNamedDimensionOverrides(std::map<winrt::hstring, uint32_t> named_dimension_overrides) {
+  named_dimension_overrides_ = named_dimension_overrides;
```

Comment:
`std::move` this :) You're making a copy when receiving by value, then another copy here

Reply:
And objects like map of string are expensive to copy needlessly

Reply:
good catch, I updated it to std::move, does it look like the correct syntax?

Reply:
Yes, that's great